flowredux/src/commonTest/kotlin/com/freeletics/flowredux/dsl/FlowReduxStateMachineTest.kt

```diff
@@ -88,4 +90,43 @@ class FlowReduxStateMachineTest {
 
         assertEquals(expectedMsg, exception.message)
     }
+
+    @Test
+    fun `observing state multiple times in parallel`() = suspendTest {
```

Comment:
Rename to something like  `observing state multiple times in parallel throws exception`() to make clear it should not be allowed to collect multiple times.

Reply:
👍 
```suggestion
    fun `observing state multiple times in parallel throws exception`() = suspendTest {
```